src/index.html

```diff
@@ -10,13 +10,116 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
     <title>Moyo header</title>
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="attr-req-value"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#index.html"
+        class="header__link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo Logo"
+          class="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              SAMSUNG
```

Comment:
[checklist](https://github.com/mate-academy/layout_moyo-header?tab=readme-ov-file#checklist): CSS is used to show all letters in Uppercase (don't type them in HTML)